test/unit/Site.test.js

```diff
@@ -0,0 +1,323 @@
+    'asset/js/vue-strap.min.js': '',
+
+    'asset/fonts/glyphicons-halflings-regular.eot': '',
+    'asset/fonts/glyphicons-halflings-regular.svg': '',
+    'asset/fonts/glyphicons-halflings-regular.ttf': '',
+    'asset/fonts/glyphicons-halflings-regular.woff': '',
+    'asset/fonts/glyphicons-halflings-regular.woff2': '',
+  };
+  fs.vol.fromJSON(json, '');
+  const site = new Site('inner/', 'inner/_site');
+  await site.generate();
+  const paths = Object.keys(fs.vol.toJSON());
+  const originalNumFiles = 13;
+  const expectedNumBuilt = 11;
+  expect(paths.length).toEqual(originalNumFiles + expectedNumBuilt);
+
+  // site
+  expect(fs.existsSync(path.resolve('inner/_site'))).toEqual(true);
+
+  // markbind
+  expect(fs.existsSync(path.resolve('inner/_site/markbind'))).toEqual(true);
+
+  // css
+  expect(fs.existsSync(path.resolve('inner/_site/markbind/css'))).toEqual(true);
+  expect(fs.existsSync(path.resolve('inner/_site/markbind/css/bootstrap.min.css'))).toEqual(true);
+  expect(fs.existsSync(path.resolve('inner/_site/markbind/css/github.min.css'))).toEqual(true);
+  expect(fs.existsSync(path.resolve('inner/_site/markbind/css/markbind.css'))).toEqual(true);
+
+  // js
+  expect(fs.existsSync(path.resolve('inner/_site/markbind/js'))).toEqual(true);
+  expect(fs.existsSync(path.resolve('inner/_site/markbind/js/setup.js'))).toEqual(true);
+  expect(fs.existsSync(path.resolve('inner/_site/markbind/js/vue.min.js'))).toEqual(true);
+  expect(fs.existsSync(path.resolve('inner/_site/markbind/js/vue-strap.min.js'))).toEqual(true);
+
+  // fonts
+  expect(fs.existsSync(path.resolve('inner/_site/markbind/fonts/glyphicons-halflings-regular.eot')))
+    .toEqual(true);
+  expect(fs.existsSync(path.resolve('inner/_site/markbind/fonts/glyphicons-halflings-regular.svg')))
+    .toEqual(true);
+  expect(fs.existsSync(path.resolve('inner/_site/markbind/fonts/glyphicons-halflings-regular.ttf')))
+    .toEqual(true);
+  expect(fs.existsSync(path.resolve('inner/_site/markbind/fonts/glyphicons-halflings-regular.woff')))
+    .toEqual(true);
+  expect(fs.existsSync(path.resolve('inner/_site/markbind/fonts/glyphicons-halflings-regular.woff2')))
+    .toEqual(true);
+});
+
+test('Site Init in existing directory generates correct assets', async () => {
+  const json = {
+    'lib/template/page.ejs': PAGE_EJS,
+  };
+  fs.vol.fromJSON(json, '');
+
+  await Site.initSite('');
+  const paths = Object.keys(fs.vol.toJSON());
+  const originalNumFiles = 1;
+  const expectedNumBuilt = 4;
+  expect(paths.length).toEqual(originalNumFiles + expectedNumBuilt);
+
+  // _boilerplates
+  expect(fs.existsSync(path.resolve('_markbind/boilerplates'))).toEqual(true);
+
+  // user defined variables
+  expect(fs.existsSync(path.resolve('_markbind/variables.md'))).toEqual(true);
+  expect(fs.readFileSync(path.resolve('_markbind/variables.md'), 'utf8')).toEqual(USER_VARIABLES_DEFAULT);
+
+  // site.json
+  expect(fs.existsSync(path.resolve('site.json'))).toEqual(true);
```

Comment:
`siteData.json` will also be generated in the future once https://github.com/MarkBind/markbind/pull/175 is merged.

Should I commit on top of your branch to add tests for this?